tasks/st1096_db_commands.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+declare PT_command
+declare PT_database
+command=$PT_command
+database=$PT_database
+
+if [ -e "/etc/sysconfig/pe-puppetserver" ] || [ -e "/etc/default/pe-puppetserver" ] || [ -e "/etc/default/puppetserver" ] || [ -e "/etc/sysconfig/puppetserver" ] # Test to confirm this is a Puppetserver
```

Comment:
This is not a valid check because we are checking for Puppet server and then running a `psql` command.  Compilers do not run postgres in any kind of infra.  It would be better to check for a running `pe-postgresql` service.

Reply:
Updated by making a PostgreSQL existence check by using ps command. Think ps is more common across different OSes?

Reply:
Sorry I didn't see this notification at first.  We don't want to parse `ps` output, the long answer why is [here](https://mywiki.wooledge.org/ProcessManagement#How_do_I_kill_a_process_by_name.3F__I_need_to_get_the_PID_out_of_ps_aux_.7C_grep_....).  For a task we should use the approach [here](https://github.com/puppetlabs/puppetlabs-pe_databases/blob/39c59706c88e9a42544fe9565efbd781d89435d2/tasks/reset_pgrepack_schema.sh#L9) of using `puppet resource`.